src/display/editor/highlight.js

```diff
@@ -407,7 +407,7 @@ class HighlightEditor extends AnnotationEditor {
       const colorPicker = (this.#colorPicker = new ColorPicker({
         editor: this,
       }));
-      return [["colorPicker", colorPicker]];
+      return [["highlightColorPicker", colorPicker]];
```

Comment:
Why is this renamed, because the ink and freetext ones are already both called `colorPicker`?